invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessRequests/AccessRequestsTab.js

```diff
@@ -0,0 +1,233 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 CERN.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component } from "react";
+import { Modal, Message, Grid, Form, Checkbox, Button } from "semantic-ui-react";
+import PropTypes from "prop-types";
+import { Formik } from "formik";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import { SuccessIcon } from "@js/invenio_communities/members";
+import {
+  RadioField,
+  TextAreaField,
+  http,
+  withCancel,
+  SelectField,
+} from "react-invenio-forms";
+import * as Yup from "yup";
+import _get from "lodash/get";
+
+export class AccessRequestsTab extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: false,
+      error: undefined,
+    };
+  }
+
+  componentWillUnmount() {
+    this.cancellableAction && this.cancellableAction.cancel();
+  }
+
+  accessRequestSchema = Yup.object({
+    allow_user_requests: Yup.bool(),
+    allow_guest_requests: Yup.bool(),
+    accept_conditions_text: Yup.string().nullable(),
+    secret_link_expiration: Yup.number(),
+  });
+
+  linkExpirationOptions = [
+    { key: 0, text: i18next.t("Never"), value: 0 },
+    { key: 30, text: i18next.t("In 1 month"), value: 30 },
+    { key: 60, text: i18next.t("In 2 months"), value: 60 },
+    { key: 180, text: i18next.t("In 6 months"), value: 180 },
+    { key: 360, text: i18next.t("In 1 year"), value: 360 },
```

Comment:
nit: i understand that 360 is the logical next value in the series here, but from a user perspective i would expect 365